load-generator/Dockerfile

```diff
@@ -1,16 +1,17 @@
-FROM gradle:6.5.0-jdk11 as builder
+FROM gradle as build
 
 # build
 WORKDIR /app
 COPY ./build.gradle ./build.gradle
 COPY ./src ./src
+COPY ./build/ ./build/
 
 RUN gradle build
-RUN tar -xvf build/distributions/app.tar
+RUN tar -xvf build/distributions/load-generator-1.0.tar
```

Comment:
can we make the version of load generator an [arg](https://docs.docker.com/engine/reference/builder/#arg)